packages/rs-dpp/src/data_contract/document_type/class_methods/try_from_schema/v0/mod.rs

```diff
@@ -71,9 +71,9 @@ impl DocumentTypeV0 {
 
         #[cfg(not(feature = "validation"))]
         if validate {
-            ProtocolError::CorruptedCodeExecution(
+            return Err(ProtocolError::CorruptedCodeExecution(
```

Comment:
Looks like this is causing issues; @QuantumExplorer: can you help me understand what is intended here? prior it was constructing the error and then discarding it

Reply:
I guess what you wrote it correct.

Reply:
it appears it's causing tests to fail? https://github.com/dashpay/platform/actions/runs/8196033514/job/22415468496?pr=1754

Reply:
@PastaPastaPasta since this seems to be a logic change, just revert this one for now and we will deal with it later. Please add a todo

Reply:
[155143c](https://github.com/dashpay/platform/pull/1754/commits/155143cf62b423753cdec05a7c25a618dcb06d16)